public/js/analytics/analyticsPage.component.js

```diff
@@ -0,0 +1,30 @@
+import angular from 'angular';
+
+export default angular.module('helloGov')
+.component('analyticsPage', {
+    template: require('./analyticsPage.html'),
+    controller: function($scope, $http, $location, constants) {
+        'ngInject';
+        // FIXME: this is super brittle to get the campaignId like this, but we're not using angular's
+        // routing so it's not possible to get it using angular yet
+        var urlSplit = $location.absUrl().split('/');
+        $scope.campaignId = urlSplit[urlSplit.length - 2];
```

Comment:
Merging this since the campaignID issue is unrelated and addressed in the other PR.